script/bootstrap

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 set -e
+source script/check_go_version
```

Comment:
I made the change you requested. We can make it more robust if someone complains. Also, I should mention that this change causes `script bootstrap` to no longer print the version of go being used (which it used to do.)  Is that okay, or should I add it back in?